daemon/container_operations_unix.go

```diff
@@ -718,6 +722,11 @@ func (daemon *Daemon) DisconnectFromNetwork(container *container.Container, n li
 	if err := container.ToDiskLocking(); err != nil {
 		return fmt.Errorf("Error saving container to disk: %v", err)
 	}
+
+	attributes := map[string]string{
+		"container": container.ID,
+	}
+	daemon.LogNetworkEventWithAttributes(n, "disconnect", attributes)
 	return nil
 }
 
```

Comment:
@aboch I think it's very weird to raise a disconnection event here when the container has not even connected to the network yet. As far as I understand this code, this is only a precaution to cleanup the sanbox before the container connects to a network, isn't it?